splice_spark2/src/test/scala/com/splicemachine/spark2/splicemachine/NativeTransformationsIT.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2012 - 2020 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package com.splicemachine.spark2.splicemachine
+
+import org.apache.spark.sql._
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{BeforeAndAfter, FunSuite, Matchers}
+
+import scala.collection.immutable.IndexedSeq
+
+@RunWith(classOf[JUnitRunner])
+class NativeTransformationsIT extends FunSuite with TestContext with BeforeAndAfter with Matchers {
```

Comment:
We can remove this IT since it doesn't really make much sense for NSDS

Reply:
Removed in new commit [2c35ac1](https://github.com/splicemachine/spliceengine/pull/3391/commits/2c35ac17df023c5b8357d22a6384464c4bb30676).